<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cam</class>
 <widget class="QMainWindow" name="Cam">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cam</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="cam1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="Id" stdset="0">
     <number>2</number>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="cam4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="Id" stdset="0">
     <number>2</number>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="pairButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>153</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Select camera and hold to pair</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cam3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="Id" stdset="0">
     <number>2</number>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="cam2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="Id" stdset="0">
     <number>2</number>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="videoLbl">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>pairButton</zorder>
   <zorder>cam1</zorder>
   <zorder>cam4</zorder>
   <zorder>cam3</zorder>
   <zorder>cam2</zorder>
   <zorder>videoLbl</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
